### DISCUSSION TITLE
Документация Node.js | Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#16] [Announcements]
# Документация Node.js | Node.js с примерами кода

Справочник web-разработчика с примерами кода - Node.js

https://nodejsdev.ru/doc/

**Comment by Dasihub** [comment]
Эта самая лучшая документация по Node js

**Reply by abdoosalomov** [reply]
Согласен

**Comment by an6re9** [comment]
Очень доступно подана информация, огромная благодарность тем, кто занимается этим ресурсом.

**Comment by stigmat-dev** [comment]
Да, все объясняется доступно. Круто!
